packages/api/app/Models/TechnologyOrderChatMessage.js

```diff
@@ -0,0 +1,10 @@
+/** @type {typeof import('@adonisjs/lucid/src/Lucid/Model')} */
+const Model = use('Model');
+
+class TechnologyOrderChatMessage extends Model {
+	static boot() {
+		super.boot();
+	}
+}
+
```

Comment:
mensagens() {
		return this.belongsToMany('App/Models/TechnologyOrderChatMessage');
	}